ShijiLighting/GHIElectronics.TinyCLR.Drivers.ShijiLighting.APA102C/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
This whole file should match the other libraries, take a look at say the SPI library itself for an example (the GUID will of course differ and should match what is in the csproj)